jarviscli/plugins/file_manager.py

```diff
@@ -0,0 +1,104 @@
+from plugin import plugin
+import os
+import shutil
+
+@plugin("file manage")
+class file_manage:
+    def __call__(self,jarvis,s):
```

Comment:
Python code style: Add space after , (multiple apperiances)

You can check python code style with:

flake8 --select E,W --max-line-length=140 --ignore E722,W503,W504,E128 PATH_TO_FILE